dynamodb_json/json_util.py

```diff
@@ -19,7 +19,7 @@ def json_serial(o):
         elif o < sys.maxsize:
             serial = int(o)
         else:
-            serial = long(o)
+            serial = int(o)
```

Comment:
This is specific for Py2.7 coverage.

Reply:
Gotcha, I will remove these  to tackle the syntaxwarnings directly then

Reply:
@Alonreznik  Come to think of it , shouldn't folks using 2.7 just use an older version of this library? And we can create a new release with this update since this isn't supported on current python versions?